### DISCUSSION TITLE
Can not set up the project in local system

### DISCUSSION THREAD

**Discussion by MihirRajChowdhury** [osmlab/name-suggestion-index#6381] [Q&A]
I want to ask can anyone help me with how to set the project up in local device as I am a college student I have never contributed to open source though i do know how to fork a repository and clone it these are some of the steps that i followed but can't run the project on my local device 
1.fork
2.clone 
3.cd name-suggestion-index
4.git checkout -b new addition 
5.npm install 
6.npm start 
but it is not starting in device can someone please help me with this as i really want to contribute to the project
and if the project requires the download of some specific apps then please mention that also

**Comment by bhousel** [comment]
Hey @MihirRajChowdhury , the name-suggestion-index project is mostly a collection of json files, so there isn’t really much to run.  Most people just edit the json files directly on GitHub, or you can edit them locally and `npm run build` the project to check them for correctness..  What sort of contribution were you looking to do?

The project wiki has a bunch of information too about what the project is and what the files do:
https://github.com/osmlab/name-suggestion-index/wiki

**Comment by WilliamJPriest** [comment]
I suggest following this guide: https://www.dataschool.io/how-to-contribute-on-github/

I just used it with no problem, you also did need to npm install or run anything, just editing json is enough
